gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -102,22 +97,54 @@ const NxBinaryDonutChartPage = () =>
           </NxTable.Row>
         </NxTable.Body>
       </NxTable>
+      <NxH3>Accessibility</NxH3>
+      <NxP>
+        <NxCode>NxBinaryDonutChart</NxCode> uses the <NxCode>meter</NxCode> role and must have an
+        accessible name. If the meter has a visible label, it is referenced by <NxCode>aria-labelledby</NxCode>
+        {' '} attribute. Otherwise, the element with role meter has a label provided by <NxCode>aria-label</NxCode>
+        {' '} attribute.
```

Comment:
The way this is phrased, saying it "has" `aria-label` or `aria-labelledby`, could be misunderstood to mean that we are somehow taking care of that automatically.  To make it clear that the caller is responsible for this, I'd change it to say "must have".

Reply:
The description is updated here 7962483aae75b6263fb0276983d8b3a14e4e9a63, thanks.

Reply:
Updated description here 0ed014303392f3c5ad34ad40b2541725f6c07c0e, thanks.